app/src/androidTest/java/com/android/solvit/provider/ui/request/RequestsDashboardScreenTest.kt

```diff
@@ -0,0 +1,175 @@
+package com.android.solvit.provider.ui.request
+
+// Jetpack Compose Testing
+import androidx.activity.ComponentActivity
+import androidx.compose.ui.test.*
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.assertIsSelected
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.android.solvit.shared.model.request.ServiceRequestRepository
+import com.android.solvit.shared.model.request.ServiceRequestStatus
+import com.android.solvit.shared.model.request.ServiceRequestViewModel
+import com.android.solvit.shared.ui.navigation.NavigationActions
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito.mock
+
+@RunWith(AndroidJUnit4::class)
+class RequestsDashboardScreenTest {
```

Comment:
I think that since the local emulator is setup and works well . It's maybe easier for you to test without mocking and using it !

Reply:
Those were old tests, I did not write them, I simply changed the name. But you are right! From now on we should use the firebase local emulator for easier testing!